dist/card.html

```diff
@@ -0,0 +1,653 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Patternfly Demo App</title>
+
+    <!-- webpack bundled css -->
+    <link rel="stylesheet" href="css/patternfly.min.css">
+    <link rel="stylesheet" href="css/patternfly-additions.min.css">
+    <style>
+      /*JENN - included styles to enable drilldown behavior to a contents/details page; only the first record demonstrates this behavior  */
+      .card-pf-view {display:block;}
+      .card-pf-view.jenn-test {pointer-events:none;}
+      .card-pf-view.jenn-test a, .card-pf-view.jenn-test input, .card-pf-view.jenn-test button {pointer-events: auto; z-index:0; position:relative;}
+
+      /*JENN - included table icon (created new class for table icon)  */
+      .fa-th-table:before {content: "\f0ce";}
```

Comment:
@jgiardino You can change this directly on the html: `<i class="fa fa-table" aria-hidden="true"></i>`